Translations/uk.po

```diff
@@ -1214,222 +1311,388 @@ msgstr ""
 "\n"
 "Впевніться що devilutionx.mpq в папці гри і актуальної версії."
 
-#: Source/diablo.cpp:972
+#: Source/diablo.cpp:1098
 msgid "Please update fonts.mpq to the latest version"
 msgstr "Будь ласка, оновіть fonts.mpq до останньої версії"
 
-#: Source/diablo.cpp:1283
+#: Source/diablo.cpp:1409
 msgid "-- Network timeout --"
 msgstr "-- Тайм-аут мережі --"
 
-#: Source/diablo.cpp:1284
+#: Source/diablo.cpp:1410
 msgid "-- Waiting for players --"
 msgstr "-- Чекаємо гравців --"
 
-#: Source/diablo.cpp:1303
+#: Source/diablo.cpp:1433
 msgid "No help available"
 msgstr "Допомога недоступна"
 
-#: Source/diablo.cpp:1304
+#: Source/diablo.cpp:1434
 msgid "while in stores"
 msgstr "в магазинах"
 
-#: Source/diablo.cpp:1446
+#: Source/diablo.cpp:1576 Source/diablo.cpp:1840
 msgid "Belt item {}"
 msgstr "Предмет з Поясу {}"
 
-#: Source/diablo.cpp:1447
+#: Source/diablo.cpp:1577 Source/diablo.cpp:1841
 msgid "Use Belt item."
 msgstr "Використати предмет з Поясу."
 
-#: Source/diablo.cpp:1462
+#: Source/diablo.cpp:1592 Source/diablo.cpp:1856
 msgid "Quick spell {}"
 msgstr "Швидкі Чари {}"
 
-#: Source/diablo.cpp:1463
+#: Source/diablo.cpp:1593 Source/diablo.cpp:1857
 msgid "Hotkey for skill or spell."
 msgstr "Гаряча клавіша для чар або таланту."
 
-#: Source/diablo.cpp:1481
+#: Source/diablo.cpp:1611 Source/diablo.cpp:1984
+msgid "Use health potion"
+msgstr "Використати зілля Зцілення"
+
+#: Source/diablo.cpp:1612 Source/diablo.cpp:1985
+msgid "Use health potions from belt."
+msgstr "Використати зілля Зцілення з пояса."
```

Comment:
This is fine - Does it fit in the game text box?

Reply:
It does
![2023-01-23_17-07-58](https://user-images.githubusercontent.com/4958520/214074926-10f19eca-dcc9-42a4-a05c-8e5670f06f14.png)